### PR TITLE
Adds ADD_TRAILING_SLASH property to AsyncContentCache

### PR DESCRIPTION
**Pull request by dkliban**
fixes: #4634

### CODE REVIEW

File: CHANGES/plugin_api/4634.feature
Comment (gerrod3) [inline CHANGES/plugin_api/4634.feature:1]: Should this be a bugfix file? Typically we don't backport features.

Comment (dkliban) [reply CHANGES/plugin_api/4634.feature:1]: sure. i'll update it.

**Review by lubosmj — COMMENTED** [review]
It is sufficient for the container plugin to override `ADD_TRAILING_SLASH` here https://github.com/pulp/pulp_container/blob/9ead8f7fb9f9e8cf06196b4086bd41f4d42b0c0d/pulp_container/app/cache.py#L21, correct?

**Comment by dkliban** [comment]
> It is sufficient for the container plugin to override `ADD_TRAILING_SLASH` here https://github.com/pulp/pulp_container/blob/9ead8f7fb9f9e8cf06196b4086bd41f4d42b0c0d/pulp_container/app/cache.py#L21, correct?

Yes, exactly. I'll update my PR in pulp_container once we have this released.

**Review by mdellweg — APPROVED** [review]

**Review by lubosmj — APPROVED** [review]

**Pull request merged**